charts/openfga/templates/deployment.yaml

```diff
@@ -148,7 +148,7 @@ spec:
 
             {{- if .Values.grpc.tls.enabled }}
             - name: OPENFGA_GRPC_TLS_ENABLED
-              value: {{ .Values.grpc.tls.enabled }}
+              value: "{{ .Values.grpc.tls.enabled }}"
```

Comment:
thank you, but maybe best to merge it in a separate PR, just to make sure we are not losing this fix if we ever need to revert this PR?

Reply:
Done, see #172